mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.erb

```diff
@@ -597,6 +599,14 @@ func TestAccContainerNodePool_withNetworkConfig(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"network_config.0.create_pod_range"},
 			},
+			// edit the updateable network config
+			{
+				Config: testAccContainerNodePool_withNetworkConfig(cluster, np, network, "TIER_UNSPECIFIED"),
```

Comment:
Have you run this locally? UNSPECIFIED values often get returned as json null (i.e. no field in the response object) and will cause a diff in Terraform. We probably need a different config w/o the `network_performance_config` block at all.

Reply:
Yes, it worked for me locally. I thought setting an empty block won't help because TF will not update these values in a case of empty block - this happened for example when I left network_config as an empty block (test failed because the node pool didn't update). Is it a bug?